sigs.yaml

```diff
@@ -0,0 +1,11 @@
+sigs:
+  - dir: sig-template
+    name: SIG Template
+    mission_statement: > 
+      Sample mission statement
+    charter_link: sig-template/charter.md
+    label: sig-template
```

Comment:
do we need this line?

Reply:
mostly took inspiration from https://github.com/kubernetes/community/blob/master/sigs.yaml, we can remove it if you want? I can make it clearer and remove the `sig` so it's `template` in the example?

Reply:
What does the label do? Is it a reference to a label in the organization that can be used for filtering issues/PRs?